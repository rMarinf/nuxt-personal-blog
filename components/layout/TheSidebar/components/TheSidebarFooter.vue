<template>
  <div class="side-menu-footer">
    <div class="mb-4">
      <h3>Subscribe for newsletter</h3>
      <!-- TODO Complete this form-->
      <form action="#" class="side-menu-subscribe-form">
        <div class="form-group d-flex">
          <div class="icon"><span class="icon-paper-plane"></span></div>
          <input
            type="text"
            class="form-control"
            placeholder="Enter Email Address"
          />
        </div>
      </form>
    </div>
    <p class="pfooter">
      Copyright &copy;
      {{ new Date().getFullYear() }}
      All rights reserved | This template is made with
      <i class="icon-heart" aria-hidden="true"></i> by
      <a href="https://colorlib.com" target="_blank">Colorlib</a>
    </p>
  </div>
</template>

<script>
export default {
  name: 'TheSidebarFooter'
}
</script>

<style scoped></style>
