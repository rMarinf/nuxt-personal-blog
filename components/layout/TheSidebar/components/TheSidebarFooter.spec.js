import { shallowMount } from '@vue/test-utils'
import TheSidebarFooter from './TheSidebarFooter.vue'

test('TheSidebarFooter', () => {
  // render the component
  const wrapper = shallowMount(TheSidebarFooter)

  expect(wrapper.element).toMatchSnapshot()
})
