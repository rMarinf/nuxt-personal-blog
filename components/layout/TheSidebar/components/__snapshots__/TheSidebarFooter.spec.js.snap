// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TheSidebarFooter 1`] = `
<div
  class="side-menu-footer"
>
  <div
    class="mb-4"
  >
    <h3>
      Subscribe for newsletter
    </h3>
     
    <form
      action="#"
      class="side-menu-subscribe-form"
    >
      <div
        class="form-group d-flex"
      >
        <div
          class="icon"
        >
          <span
            class="icon-paper-plane"
          />
        </div>
         
        <input
          class="form-control"
          placeholder="Enter Email Address"
          type="text"
        />
      </div>
    </form>
  </div>
   
  <p
    class="pfooter"
  >
    
    Copyright Â©
    2020
    All rights reserved | This template is made with
    
    <i
      aria-hidden="true"
      class="icon-heart"
    />
     by
    
    <a
      href="https://colorlib.com"
      target="_blank"
    >
      Colorlib
    </a>
  </p>
</div>
`;
